common/po/messages.pot

```diff
@@ -950,7 +950,7 @@ msgid "Changes"
 msgstr ""
 
 #: ../../qt/logviewdialog.py:100
-msgid "Informations"
+msgid "Information"
```

Comment:
Pretty sure that this would mess up translations.

Reply:
"mess up" is a bit to much. Yes, it will modify the translation and do open the need to translate that part again.

But we do this in the `dev` branch and the next release is far away. There is enough time to update the translation. We still do have some work todo with the translation and the related workflow. I see no need to make this a part of that PR.

Reply:
Okay, I think I understand. If you know what's going on here, I'm fine with following your plan. I'd just like to avoid leaving this "hanging", for fear that it might come back to "bite" us later when we've forgotten about this change ;)

Reply:
It will just appear as a new & untranslated string in the translation frontend (currently launchpad).